.github/workflows/swift-toolchain.yml

```diff
@@ -27,6 +25,24 @@ on:
         description: 'Code Sign'
         default: false
         type: boolean
+
+      default_runner:
+        description: 'The default runner to use for jobs in this workflow'
+        default: 'windows-latest'
+        required: false
+        type: string
+
+      compilers_runner:
+        description: 'The runner to use for building the compilers in this worflow. If unset, uses default_runner'
+        type: string
+        required: false
+
+      dry_run:
```

Comment:
`publish_artifacts` would be clearer a name

Reply:
Done in https://github.com/compnerd/swift-build/pull/747. I'll delete and rebase once that's merged